java/vector/src/main/java/org/apache/arrow/vector/complex/ListVector.java

```diff
@@ -55,14 +58,12 @@ public static ListVector empty(String name, BufferAllocator allocator) {
     return new ListVector(name, allocator, FieldType.nullable(ArrowType.List.INSTANCE), null);
   }
 
-  final UInt4Vector offsets;
-  final BitVector bits;
-  private final List<BufferBacked> innerVectors;
-  private Mutator mutator = new Mutator();
-  private Accessor accessor = new Accessor();
+  private ArrowBuf validityBuffer;
```

Comment:
Do we want to move this to base class?

Reply:
Validity Buffer is a property of ListVector and probably not of BaseRepeatedValueVector. At least that is how it is today. That is why didn't move it to base class.

Reply:
Ok. This is fine.